templates/default/entity/Entry/edit.tpl.php

```diff
@@ -20,6 +20,18 @@
 
     /* @var \Idno\Core\Template $this */
 
+?>
+<?php
+if (!empty($vars['object']->inreplyto)) {
```

Comment:
consider ternary operations here too for clarity.

Reply:
Like this:
`$vars['object']->inreplyto = (!is_array($vars['object']->inreplyto)) ? array($vars['object']->inreplyto): array();`